privacy.adoc

```diff
@@ -284,45 +284,45 @@ into a centralized permissioned system where you have an “IOU” from
 the blacklist providers.
 ____
 
-It seems privacy and fungibility go hand-in-hand. Fungibility will
-weaken if privacy is weak, for example, because coins from unwanted
-persons can become blacklisted.  Privacy will weaken if fungibility is
-weak since you have to ask the blacklist providers about which coins
-to accept, possibly revealing your IP address, email address, and
-other sensitive information. They're so intertwined that it's hard to
-talk about them in isolation.
+It seems that privacy and fungibility go hand-in-hand. Fungibility will
+weaken if privacy is weak, for example as coins from unwanted
+people may become blacklisted. In the same way, privacy will weaken if fungibility is
+weak: if there is a blacklist, you will have to ask the blacklist providers about which coins
+to accept, thereby possibly revealing your IP address, email address, and
+other sensitive information. These two features are so intertwined that it's hard to
+talk about either of them in isolation.
 
 [[privacymeasures]]
 === Privacy measures
 
-Several techniques have been developed that help protect from privacy
-leaks. Among the most obvious ones are, as
+Several techniques have been developed to help people protecting themselves from privacy
+leaks. Among the most obvious ones is, as
 <<satoshi-unique-addresses,noted by Nakamoto>> above, using unique
-addresses for all transactions, but several others exist. We're not
+addresses for every transaction, but several others exist. We're not
 going to teach you how to become a privacy ninja. However, Bitcoin Q+A has
-a quick summary of privacy-enhancing technology, somewhat ordered by
+a quick summary of privacy-enhancing technologies, somewhat ordered by
 how hard they are to implement, at
-https://bitcoiner.guide/privacytips/. You'll notice when you read this
-that Bitcoin privacy often has to do with stuff outside of Bitcoin;
-for example: don't brag about your bitcoins, and use Tor and VPN. The
-post also lists some things directly related to Bitcoin:
+https://bitcoiner.guide/privacytips/. When you read it, you'll notice
+that Bitcoin privacy often has to do with stuff outside of Bitcoin.
+For example, you shouldn't brag about your bitcoins, and you should always use Tor and VPN. The
+post also lists some measures directly related to Bitcoin:
 
 Full node:: If you don't use your own full node, you will leak lots of
 information about your wallet to servers on the internet. Running a
 full node is a great first step.
 
 Lightning Network:: Several protocols exist on top of Bitcoin, for
-example, the Lightning Network and Blockstream's Liquid sidechain.
+example the Lightning Network and Blockstream's Liquid sidechain.
 
 CoinJoin:: A way for multiple people to merge their transactions into
-one, making it harder to do address clustering.
+one, making it harder to do address clustering [comm: not chain analysis?].
```

Comment:
Yes, we can use chain analysis, but we haven't introduced that term (as far as I recall). If you prefer chain analysis, go for it.

On the other hand, readers are expected to know how Bitcoin works, so chain analysis could be pretty self-expanatory.

Reply:
I think "chain analysis" is fairly intuitive to understand, especially for the readers of this book who won't be complete newbies.
My doubt is probably a consequence of my own ignorance: are chain analysis and address clustering the same thing? If so, we can definitely keep the original version, I don't have any specific reason to prefer "chain analysis" instead

Reply:
Chain analysis is a broader term than address clustering. I agree, we can keep "chain analysis".